boards/imix/src/main.rs

```diff
@@ -467,10 +479,19 @@ pub unsafe fn reset_handler() {
         ninedof: ninedof,
         radio_driver: radio_driver,
         usb_driver: usb_driver,
+        nrf51822: nrf_serialization,
     };
 
     let mut chip = sam4l::chip::Sam4l::new();
 
+    // Need to reset the nRF on boot
+    sam4l::gpio::PB[07].enable();
+    sam4l::gpio::PB[07].enable_output();
+    sam4l::gpio::PB[07].clear();
+    sam4l::gpio::PB[07].set();
```

Comment:
Is there any delay / wait / bootup time necessary here?

Is there a hold time needed for the reset pin (or are we just counting on this bit-bangy-ish approach being slow enough?)

Reply:
I literally copied this from `hail`... i think we're just counting on it being slow enough, but it's hooked up to the reset pin, so should be ok?

Reply:
![image](https://user-images.githubusercontent.com/339422/31965432-3efbfdbe-b8bc-11e7-98cc-2de5c33344eb.png)

I suppose we're probably not going to be faster than 200ns in practice, may be worth a comment to the effect of `// minimum hold time is 200 ns, so no delay needed here` so that an other hypothetical platform knows the assumption this code makes

Reply:
done

Reply:
@brghena is looking this up, it's ~20ns / instruction and ~40ns rise/fall time on pins on the SAM4L, so there's a universe where this happens too fast. This _may_ be accounting for some of the nrf flakiness?

It costs very little to a tiny delay loop here, may be worth it

Reply:
added a 10 iteration nop loop